Inspicio/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -107,9 +107,11 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<string>("Description");
 
-                    b.Property<int>("NoOfDislikes");
+                    b.Property<int>("NoOfApprovals");
```

Comment:
Would you be able to explain this in more detail? I think I'm misunderstanding how we would be able to calculate it without storing anything?